tests/tests/test-balance.ts

```diff
@@ -26,7 +26,7 @@ describeWithMoonbeam("Moonbeam RPC (Balance)", `simple-specs.json`, (context) =>
       {
         from: GENESIS_ACCOUNT,
         to: TEST_ACCOUNT,
-        value: "0x200", // Must me higher than ExistentialDeposit (500)
+        value: "0x200", // Must be higher than ExistentialDeposit (0)
```

Comment:
So there is no minimum anymore?

Reply:
Yes, it's been that way for a while. See https://github.com/PureStake/moonbeam/pull/119

Reply:
good to know thanks